content/python/concepts/functions/terms/isinstance.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'isinstance()'
+Description: 'Returns True if the given object is the specified type. Otherwise the function will return False.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Algorithms'
+  - 'Booleans'
+CatalogContent:
+- 'learn-python-3'
+- 'paths/computer-science'
+---
+
+The **isinstance()** function determines whether one object is a designated value type. If it is, the function will return `True`, otherwise the function will return `False`.
+
+## Syntax
+
+```py
+isinstance(object, class)
+```
+
+The example below demonstrates the use of the `isinstance()` method to test if multiple objects are of type `str`:
```

Comment:
Should go after the Examle header.

Please include the available types and link out to the data type entry on this line instead, in reference to the syntax block above.

Reply:
To clarify, you're suggesting I move this section underneath the Example header, leaving the Syntax header section empty? Also what is the best place to link out the data types? Within the Codecademy docs repo?

Reply:
Yes, and to fill the syntax header with the relevant info. The syntax section can have associated text that links out to different [data types](https://www.codecademy.com/resources/docs/python/data-types) like [strings](https://www.codecademy.com/resources/docs/python/strings)

Reply:
Okay, I'm still a little confused on if the hierarchy of the .md is set up to what you intended but I committed my changes based on your feedback and can revise further as needed!